### ISSUE TITLE
Soundcloud plays only the first song

### ISSUE THREAD

**Issue by AConsolePeasant** [matthieugrieger/mumbledj#176]
#### Before submitting this issue, please acknowledge that you have done the following:
- [x] I've at least skimmed through the [README](https://github.com/matthieugrieger/mumbledj/blob/master/README.md)
- [x] I have checked that I am running the latest version of MumbleDJ (use `mumbledj --version` when starting the bot or use the MumbleDJ version command in Mumble)
- [x] I have [searched through the existing issues](https://github.com/matthieugrieger/mumbledj/issues) to see if my issue has been answered already

---

#### What type of issue is this?
- [x] Bug report (encountered problems with MumbleDJ)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other:

---

#### Log output of bot with `--debug` flag (likely only for bug reports):

---

#### Description of your issue:
I've reinstalled MumbleDJ after the most recent bugfix, But I think I've found a bug with soundcloud support. I'm not sure if this is due to cache, but adding more than one soundcloud track will only download* the first song, and play it for every other track from soundcloud. I haven't tried disabling cache, as I'm currently away, and can't access the server.

**Comment by matthieugrieger** [comment]
Good catch. It is related to caching, fixing it now...

**Comment by matthieugrieger** [comment]
This should be fixed now. Here's what was happening:

The file stored on disk for SoundCloud songs is of the form `track_id.track`. I made a mistake in the code for parsing API responses for SoundCloud, which resulted in empty strings for the track id. Therefore, all SoundCloud tracks were given the filename `.track`. I fixed the API response parsing, so each track should now have a unique id.

Let me know if you still experience any issues!
